### DISCUSSION TITLE
"Next block" fee distribution graph

### DISCUSSION THREAD

**Discussion by Ben-in-Chicago** [mempool/mempool#3642] [Q&A] [answered]
Hello! Can someone explain what the values on the x-axis represent when viewing the fee distribution graph of the next block (https://mempool.space/mempool-block/0) ?

![image](https://user-images.githubusercontent.com/99691112/229873351-671a3f67-b684-430e-9947-327bd0107551.png)

Thanks
Ben

**Comment by softsimon** [comment, marked as answer]
Nothing actually. It's just an index number of the different data points.. 0 to 6.
